x-pack/plugins/enterprise_search/server/routes/app_search/engines.test.ts

```diff
@@ -71,7 +71,8 @@ describe('engine routes', () => {
       it('should return 404 with a message', async () => {
         await mockRouter.callRoute(mockRequest);
 
-        expect(mockRouter.response.notFound).toHaveBeenCalledWith({
+        expect(mockRouter.response.customError).toHaveBeenCalledWith({
```

Comment:
Should these tests assertions no longer say `return 404` since they return `502` now?

Reply:
Good catch! As an FYI we'll likely remove/simplify these tests once Jason's PR is in (#75487), since the helper will take care of catching errors.